Makefile

```diff
@@ -3,7 +3,7 @@ CFLAGS+=-g -O0 -fPIC -Wall
 LDFLAGS+=
 INCDIRS+=-I. -Isrc
 
-SOURCES=src/acvp.c src/acvp_aes.c src/acvp_des.c src/acvp_hash.c src/acvp_transport.c src/acvp_util.c src/parson.c 
```

Comment:
General comment: Did you get resolution from Apostol about the questions you had ?